### DISCUSSION TITLE
GitHub Pages baseURL: trailing slash or no?

### DISCUSSION THREAD

**Discussion by jamesbraza** [thegeeklab/hugo-geekdoc#642] [Q&A] [answered]
I am going back and forth on this, when using GitHub Pages, should `baseURL` have:
- No trailing slash: `baseURL = "https://user.github.io/repo"`
- Trailing slash: `baseURL = "https://user.github.io/repo/"`

I see [this `config.toml`](https://github.com/Alex-Space/btxwiki/blob/7a7b943de391a8844044f10d5d67b607d7909cfa/config.toml#L2) uses no trailing slash

**Comment by xoxys** [comment]
Please take a look at https://geekdocs.de/usage/getting-started/#subdirectories

**Reply by xoxys** [reply, marked as answer]
Nevertheless, both variants should work in general.

**Reply by jamesbraza** [reply]
Thanks for sharing that link, yeah I had previously read though it.

Based on GitHub's Hugo starter workflow having [this trailing slash here](https://github.com/actions/starter-workflows/blob/0582bb6eb45cd5c60bcb0ce2a35fe50bbc0c2c0d/pages/hugo.yml#L59), it made me think I should _not_ have the trailing slash in `baseURL`.  However, the subdirectories link above uses a trailing slash, so I decided to add one to match the docs here.
